lib/rubocop/cop/rspec/be_eql.rb

```diff
@@ -45,7 +45,7 @@ class BeEql < Base
 
         # @!method eql_type_with_identity(node)
         def_node_matcher :eql_type_with_identity, <<~PATTERN
-          (send _ :to $(send nil? :eql {true false int float sym nil}))
+          (send _ :to $(send nil? :eql {boolean int float sym nil}))
```

Comment:
```suggestion
          (send _ :to $(send nil? :eql {boolean numeric sym nil}))
```
This needs to be checked though, and the doc above needs to be adjusted.

I don’t mind leaving this out

Reply:
`numeric` is more than `int` and `float`, so up to you if you want to change behaviour. I was only making equivalent changes 😄